code/Employee.php

```diff
@@ -89,10 +89,18 @@ class Employee extends DataObject
 
     private static $table_name = 'Employee';
 
+    /**
+     * Use basic scaffold settings (no tabs, etc)
+     */
+    private static array $scaffold_cms_fields_settings = [
```

Comment:
These settings are functionally equivalent to that happens when `$this->scaffoldFormFields();` is call with no params